Source/Lib/Fluxor.Blazor.Web/Fluxor.Blazor.Web.csproj

```diff
@@ -30,6 +30,7 @@
 						<Pack>True</Pack>
 						<PackagePath></PackagePath>
 				</None>
+				<InternalsVisibleTo Include="Fluxor.Blazor.Web.UnitTests" />
```

Comment:
I didn't know about this :)

Reply:
If you always take this long to review PRs... 🤪